source/includes/usage-examples/sample-app-intro.rst

```diff
@@ -4,6 +4,6 @@ Sample Application
 You can use the following sample application to test the code examples on this
 page. To use the sample application, perform the following steps:
 
-1. Ensure you have the {+driver-short+} installed.
-#. Copy the following code and paste it into a new ``.cpp`` file.
-#. Copy a code example from this page and paste it on the specified lines in the file.
\ No newline at end of file
+1. Ensure you have the {+driver-short+} installed in a location from which your project can import it.
+#. Copy the following code and paste it into a new ``.cpp`` file within your project.
+#. Copy a code example from this page and paste it within the highlighted section of the file.
```

Comment:
The highlighted lines are no longer correct after modifications to the source file changed the line count.

Reply:
Fixed